src/index.scss

```diff
@@ -8,3 +9,7 @@ The React <Content /> component uses the `main` element which IE11 doesn't recog
 main {
   display: block;
 }
+
+$feature-flags: (
```

Comment:
Per https://www.carbondesignsystem.com/tutorial/react/step-1#add-ui-shell-sass, the feature -flags mapping should come before the styles import.